pynautobot/core/response.py

```diff
@@ -205,7 +206,7 @@ def __str__(self):
         return getattr(self, "display", None) or getattr(self, "name", None) or getattr(self, "label", None) or ""
 
     def __repr__(self):
-        return str(self)
+        return "<{}.{} name='{}' at {}>".format(self.__class__.__module__, self.__class__.__name__, self, hex(id(self)))
```

Comment:
```suggestion
        return "<{}.{} ('{}') at {}>".format(self.__class__.__module__, self.__class__.__name__, self, hex(id(self)))
```

I wonder if using `name=` here could be confusing if there isn't a `name` field on the model. Seeing as the `__str__` method returns display, name or label (or an empty string) it may be disingenuous to claim it is always the "name". Simply wrapping it in parentheses seems a happy medium to me unless you want to dynamically change the label to match the attr being returned.

Reply:
@joewesch done!
___
Now it looks like this:
```
<pynautobot.models.dcim.Devices ('hAP lite router') at 0x2bae6948f40>
```